### PR TITLE
Changes for versioning

### PR DESCRIPTION
**Pull request by dleeatosisoft**

### CODE REVIEW

File: Documentation/DataIngress/OMF_Ingress_Subscriptions.md [hunk 1]
```diff
@@ -31,86 +23,46 @@ Subscription information is contained in an object called Subscription which has
...
-
-``tenantId``
-  Unique Id for the tenant. 
-``namespaceId``
-  Unique Id for the namespace.   
-
-**Returns**
-
-An integer count of subscriptions. 
+| Enabled              | boolean                 | Whether the topic exists or not.        |
```
Comment (sagarwal29) [inline Documentation/DataIngress/OMF_Ingress_Subscriptions.md]: T in 'topic' should be capital

Comment (dleeatosisoft) [reply Documentation/DataIngress/OMF_Ingress_Subscriptions.md]: Good catch. I fixed it.

File: Documentation/DataIngress/OMF_Ingress_Subscriptions.md [hunk 2]
```diff
@@ -31,86 +23,46 @@ Subscription information is contained in an object called Subscription which has
...
 | TopicId              | string                  | Unique Id for the Topic we are subscribing to. |
 | TopicTenantId        | string                  | Identifies the owner of the Topic.     |
 | TopicNamespaceId     | string                  | Identifies the namespace for the Topic |
 | TenantId             | string                  | Identifies the owner of the Subscription. |
-| NamespaceId          | string                  | Identifies the namespace for the Subscription |
-| IsRevoked            | boolean                 | Revocation status of the Subscription. |
+| NamespaceId          | string                  | Identifies the namespace for the Subscription. |
 | Description          | string                  | Description of the Subscription.       |
-| Type                 | integer                 | An enumeration where OCSDataStore=1, Standard=2  |
+| Type                 | integer                 | An enumeration which describes the type of Subscription where OCSDataStore=1 |
```
Comment (mschellenger) [inline Documentation/DataIngress/OMF_Ingress_Subscriptions.md]: Change to SDS here

File: Documentation/DataIngress/OMF_Ingress_Topics.md [hunk 3]
```diff
@@ -217,23 +198,22 @@ Updates an existing topic. Only the topic name and description can be updated.
 ``namespaceId``
   Unique Id for the namespace. 
 ``topicId``
-  Topic Id for the topic to be updated.
+  Topic Id for the Topic to be updated.
```
Comment (mschellenger) [inline Documentation/DataIngress/OMF_Ingress_Topics.md:201]: Not exactly sure how to do this, but can you try to find a way to document that you need the object ID in the body and the route for a PUT call to a topic or subscription, but it is not allowed in the POST call?

**Review by mschellenger — APPROVED** [review]

File: Documentation/DataIngress/OMF_Ingress_Subscriptions.md (hunk 2 above)
Comment (dleeatosisoft) [reply Documentation/DataIngress/OMF_Ingress_Subscriptions.md]: Fixed.

File: Documentation/DataIngress/OMF_Ingress_Topics.md (hunk 3 above)
Comment (dleeatosisoft) [reply Documentation/DataIngress/OMF_Ingress_Topics.md:201]: Fixed.

**Pull request merged**